packages/tools/devtools/devtools-core/README.md

```diff
@@ -29,10 +29,56 @@ The Devtools' API surface is designed to fit nicely into most application flows.
 ### Initialization
 
 To initialize a devtools session for your container, call `initializeDevtools`.
-This function accepts a `DevtoolsLogger` for recording and communicating telemetry data, a list of initial Fluid `Containers` to associate with the session, and (optionally) customized data visualization configurations for visualizing `Container` data.
+This function accepts a `DevtoolsLogger` for receiving Fluid telemetry from your application, a list of initial Fluid
+`Containers` to associate with the session, and (optionally) customized data visualization configurations for visualizing
+`Container` data.
 
 TODO: link to API docs once API shape has settled.
 
+#### About the `DevtoolsLogger`
+
+The `DevtoolsLogger` is an optional piece when calling `initializeDevtools` but it is strongly recommended that you use
+it because several features in Fluid Devtools are powered by the telemetry that Fluid Framework generates, and this
+logger is the way in which that telemetry gets into Fluid Devtools.
+
+You can obtain a `DevtoolsLogger` by calling its constructor, and then should pass it to `initializeDevtools` _and_
+to the `Loader` instance you'll use to load Containers as shown below.
+This way the logger will receive all the telemetry generated by Fluid Framework in your application, and forward it to
+Fluid Devtools as necessary.
+
+```typescript
+import { DevtoolsLogger, initializeDevtools } from "@fluid-experimental/devtools-core";
+
+const devtoolsLogger = new DevtoolsLogger();
+const devtools = initializeDevtools({ logger: devtoolsLogger });
+
+const loader = new Loader({
+    logger: devtoolsLogger,
+    // Other necessary properties ...
+});
+```
+
+If you're working with `AzureClient` instead of lower-level APIs like the one described above, you probably want to
+refer to the [@fluidframework/devtools package](../devtools/) instead of this one.
```

Comment:
Relative file paths like these will work fine in the context of the repo, but these READMEs also get published alongside the packages to npm, where the relative links won't work. We should probably use explicit github links instead.

Reply:
Either that or link to the relevant npm page. That page will have a link back to the github anyways.

Reply:
I was a bit torn which of the two to use. I liked the github links, but they're more prone to breaking if we move things around. The npm ones are more stable, but require another click (or more) to go back to the code for the package; which might not be the main use case of the links in the README so that's probably ok. And those will need to be updated when we move from the `@fluid-experimental` scope. But it felt like the way to go.